src/language/providers/attributeCompletion.ts

```diff
@@ -18,13 +18,14 @@
 import * as vscode from 'vscode'
 
 import {
+  _XmlItem,
```

Comment:
Names that begin with _ are not typically to be used publicly.  This doesn't feel right to me.